app/grandchallenge/evaluation/models.py

```diff
@@ -1403,6 +1407,35 @@ def metrics_json_file(self):
             if output.interface.slug == "metrics-json-file":
                 return output.value
 
+    @property
+    def missing_metrics_json_file(self):
```

Comment:
To me this property name suggests that the metrics.json file is missing. Maybe rename to just `missing_metrics`? or if you want it to be more descriptive `metrics_missing_from_metrics_file` or something?

Reply:
I see your point. I meant to emphasize that missing metrics here are for json file component interface only. But probably better to rename it to missing_metrics.